src/main/java/edu/ucsb/cs156/example/repositories/RecommendationRequestRepository.java

```diff
@@ -0,0 +1,15 @@
+package edu.ucsb.cs156.example.repositories;
+
+import edu.ucsb.cs156.example.entities.RecommendationRequest;
+
+import org.springframework.data.repository.CrudRepository;
+import org.springframework.stereotype.Repository;
+
+/**
+ * The UCSBDateRepository is a repository for UCSBDate entities.
```

Comment:
I think the comment was meant to say RecommendationRequestRepository and RecommendationRequest.